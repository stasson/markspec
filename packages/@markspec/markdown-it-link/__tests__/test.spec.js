const  MarkdownIt = require('markdown-it')
const Plugin = require('..');
const fs = require('fs');
const path = require('path');

const md = new MarkdownIt().use(Plugin);
const fixture = fs.readFileSync( path.join(__dirname, 'fixture.md'),'utf8')

describe('markdown-it-link', () => {
    it('renders graphs',  ()=> {
        const result = (md.render(fixture))
        // fs.writeFileSync(path.join(__dirname, 'rendering.html'),result,'utf8')
        expect(md.render(fixture)).toMatchSnapshot()
    });
});
