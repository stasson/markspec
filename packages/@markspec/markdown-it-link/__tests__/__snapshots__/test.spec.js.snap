// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`markdown-it-link renders graphs 1`] = `
"<h1>Links</h1>
<ul>
<li><a href=\\"https://github.com/\\" target=\\"_blank\\">source</a></li>
<li><a href=\\"https://github.com/\\" target=\\"_blank\\">https://github.com/</a></li>
<li><a href=\\"../../index.html\\">source</a></li>
<li><a href=\\"../../index.html\\">readme</a></li>
<li><a href=\\"../../index.html#link\\">readme</a></li>
<li><a href=\\"./fixture.html\\">fixture</a></li>
<li><a href=\\"./fixture.html#links\\">home</a></li>
</ul>
"
`;
